src/Microsoft.Diagnostics.Monitoring.Options/EgressCategoryOptions.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Diagnostics.Monitoring.WebApi;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Text.Json.Serialization;
+
+namespace Microsoft.Diagnostics.Tools.Monitor.Egress
+{
+    /// <summary>
+    /// Egress provider options for external egress providers.
+    /// </summary>
+    internal sealed partial class EgressCategoryOptions
```

Comment:
Only use `partial` when it has actual usage. This is happening in several locations. Otherwise I look through the code trying to find the other part of the class and don't find it.